<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selector</class>
 <widget class="QMainWindow" name="Selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>540</width>
      <height>420</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>0</y>
       <width>500</width>
       <height>418</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionStartSelection"/>
   <addaction name="actionCalcMorph"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomAuto"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open image</string>
   </property>
  </action>
  <action name="actionStartSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>startSelection</string>
   </property>
   <property name="toolTip">
    <string>start selection of area</string>
   </property>
  </action>
  <action name="actionCalcMorph">
   <property name="text">
    <string>calcMorph</string>
   </property>
   <property name="toolTip">
    <string>calculate morpth things</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="toolTip">
    <string>zoom in</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="toolTip">
    <string>zoom out</string>
   </property>
  </action>
  <action name="actionZoomAuto">
   <property name="text">
    <string>ZoomAuto</string>
   </property>
   <property name="toolTip">
    <string>zoom auto</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
